server.py

```diff
@@ -12,6 +16,7 @@
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 app = Flask(__name__)
+metrics = PrometheusMetrics(app)
```

Comment:
This we are exporting only default metrics, right ? @malmanzor

Reply:
Yes, only the defaults